src/index.html

```diff
@@ -22,6 +22,308 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <nav class="nav">
+        <div class="nav__img">
+          <img
+            src="./images/logo.png"
+            alt="Logo"
+          />
+        </div>
+        <div class="nav__container">
+          <div class="nav__items">APPLE</div>
+          <div class="nav__items">SAMSUNG</div>
+          <div class="nav__items">SMARTPHONES</div>
+          <div data-qa="nav-hover" class="nav__items">LAPTOPS & COMPUTERS</div>
+          <div class="nav__items">GADGETS</div>
+          <div class="nav__items">PHOTO</div>
+          <div class="nav__items">VIDEO</div>
+        </div>
```

Comment:
nav__item elements should be links. use semantics instead of just divs. It's a good practice to create navigation using list (ul, li)